### PR TITLE
done

### PR DESCRIPTION
**Pull request by antonver**

### CODE REVIEW

**Comment by antonver** [comment]
Снимок экрана 2024-05-04 в 18 35 48
I wanted to ask why I do not have Driverse, but have users

**Review by vsmutok — CHANGES_REQUESTED** [review]
Please remove `db.sqlite3` and `.idea` from PR

File: taxi/models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
```
Comment (Polyakiv-Andrey) [inline taxi/models.py:7]: add str method for model

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name = models.CharField(max_length=256, unique=True)
+    country = models.CharField(max_length=256)
+
+
+class Driver(AbstractUser):
```
Comment (Polyakiv-Andrey) [inline taxi/models.py:12]: add str method for model

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    license_number = models.CharField(max_length=256, unique=True)
+
+
+class Car(models.Model):
```
Comment (Polyakiv-Andrey) [inline taxi/models.py:16]: add str method for model

File: taxi/__init__.py
Comment (Polyakiv-Andrey) [inline taxi/__init__.py:1]: delete all pyc files from git

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**